pengwin-setup.d/ruby.sh

```diff
@@ -56,6 +56,8 @@ if (confirm --title "RUBY" --yesno "Would you like to download and install Ruby
   echo "Rehashing rbenv"
   rbenv rehash
 
+  bash "${SetupDir}"/jetbrains_support.sh --yes "$@"
```

Comment:
Change here for dashes

Reply:
Done. 
I changed the file name and all the reference